src/template.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
+    />
+    <script
+      defer
+      src="https://kit.fontawesome.com/667d7efdcc.js"
+      crossorigin="anonymous"
+    ></script>
+
+    <title>Movie Slider</title>
+  </head>
+  <body>
+    <div class="container">
+      <div class="navbar">
+        <h2 class="logo">Movie concert</h2>
+        <ul class="nav-list">
+          <li class="nav-item movies">Movies</li>
+          <li class="nav-item">TV</li>
+          <li class="nav-item">About</li>
+        </ul>
```

Comment:
- [ ]  Your project should contain Home page Header similar  with the one from the provided mock up.  Your  navigations and logo are aligned left and right respectively.  Which is different the mockup. 
Your header 
![image](https://github.com/husush23/capstone--2/assets/43877689/25ad1345-49bd-4f12-b0e8-ef8f59f850c3)

mockup header 
![image](https://github.com/husush23/capstone--2/assets/43877689/b32dae12-d2b9-447d-988d-4fb7120a8172)